integration_tests/integration/prison-register-list.spec.ts

```diff
@@ -100,34 +100,88 @@ context('Prison register - prison list navigation', () => {
     // Cancel the text search filter by clicking on the filter tag
     page.cancelTextSearchFilter('Albany').click()
     page.showFilterButton().click()
-    // Check we are now only filtered on prisons
+    // Check we are not filtering on prisons
     page.cancelTextSearchFilter('Albany').should('not.exist')
     page.textSearchFilter().should('be.empty')
     cy.url().should('not.contain', 'textSearch=Albany')
   })
 
+  it('Will change the gender filter', () => {
+    IndexPage.verifyOnPage().prisonRegisterLink().click()
+    const page = AllPrisons.verifyOnPage()
+
+    // Check the filter defaults to all prisons
+    page.showFilterButton().click()
+    page.maleFilter().should('have.attr', 'type', 'checkbox').should('be.checked')
+    page.femaleFilter().should('have.attr', 'type', 'checkbox').should('be.checked')
+    // Filter on female prisons only
+    page.maleFilter().click()
+    page.applyFilterButton().click()
+    page.showFilterButton().click()
+    // Check the female prisons filter has been applied
+    page.maleFilter().should('not.be.checked')
+    page.femaleFilter().should('be.checked')
+    cy.url().should('include', 'genders=FEMALE')
+    // Add back in male prisons
+    page.maleFilter().click()
+    page.applyFilterButton().click()
+    page.showFilterButton().click()
+    // Check the filter has reset
+    page.maleFilter().should('be.checked')
+    page.femaleFilter().should('be.checked')
+    // Filter on male prisons only
+    page.femaleFilter().click()
+    page.applyFilterButton().click()
+    page.showFilterButton().click()
+    // Check the male prisons filter has been applied
+    page.maleFilter().should('be.checked')
+    page.femaleFilter().should('not.be.checked')
+    cy.url().should('include', 'genders=MALE')
+  })
+
```

Comment:
Should we test for deselecting male and female and test filter values returned.

Reply:
Good shout. Added that to `Will change the gender filter`